pimd/pim_nb_config.c

```diff
@@ -1504,11 +1504,18 @@ int routing_control_plane_protocols_control_plane_protocol_pim_address_family_re
  */
 int lib_interface_pim_address_family_create(struct nb_cb_create_args *args)
 {
+	struct interface *ifp;
+
 	switch (args->event) {
 	case NB_EV_VALIDATE:
 	case NB_EV_PREPARE:
 	case NB_EV_ABORT:
 	case NB_EV_APPLY:
+		ifp = nb_running_get_entry(args->dnode, NULL, true);
```

Comment:
Shouldn't this be under a validation state?

Reply:
validation to me states that it's a legal config, this implies that we should not be creating state under validation state at all, correct?

Reply:
>validation to me states that it's a legal config

Yes, that's why I asked if this shouldn't be in a validation state, while pim_if_new() inside PREPARE/APPLY.

Reply:
it definatly should not be created during abort as well :)